incubator/cassandra/values.yaml

```diff
@@ -174,9 +174,13 @@ backup:
   # Destination to store the backup artifacts
   # Currently only s3 is supported
   # Additional support can added.
-  # Ref: https://github.com/maorfr/skbn
+  # Ref: https://github.com/nuvo/skbn
   destination: s3://bucket/cassandra
 
+  # Copy n files in parallel
+  # 0 for full parallelism
+  parallel: 1
```

Comment:
Don't we want to maintain default ?

Reply:
I always feel that the `values.yaml` file should provide an easy interface to change things when you need, instead of digging in the templates...
when increasing parallelism - it increases resource usage, so I think it is better left for the user to decide.

What do you think?